policyengine_us/variables/gov/ssa/ssi/eligibility/income/ssi_income_eligible.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ssi_income_eligible(Variable):
```

Comment:
Let's flip it to indicate SGA, and also disambiguate from the separate income limit (not exactly a limit but the income level at which SSI would fall to zero through the phase-out). Also we'll include ssi in the name to distinguish from SSDI's definition, which applies to blind people.
```suggestion
class ssi_engaged_in_sga(Variable):
```

Reply:
not quite what i suggested. let's make this income > sga